upcoming-release-notes/789.md

```diff
@@ -0,0 +1,6 @@
+---
+category: Feature
```

Comment:
I would personally tag this as Maintenance since it won’t affect end users

Reply:
I'm ok with that :)